windows/utils/win_enable_vmtools_log.yml

```diff
@@ -7,6 +7,11 @@
 - name: "Check and create VMware Tools config file"
   include_tasks: win_check_create_vmtools_conf.yml
 
+- name: "Set fact of collecting vmtools log file"
```

Comment:
Task name can be changed to `Set facts of VMware Tools log files`

Reply:
Done.